db/zm_update-1.33.12.sql

```diff
@@ -0,0 +1,6 @@
+--
+-- Add primary keys for Logs and Stats tables
+--
+
```

Comment:
close but we need these scripts to be able to be re-applied without failure, check other scripts for how to test for existence in the table first.

Reply:
I see, I think e35c165 should correct this.

Reply:
And e5104ac adds to actually prepare&execute the statements, oops.